OMCompiler/Compiler/Template/CodegenUtil.tpl

```diff
@@ -173,8 +173,7 @@ template crefCComment(SimVar v)
 ::=
   match v
   case SIMVAR(isProtected = true, source = SOURCE(info = SOURCEINFO(fileName = fileName))) then
-    if not boolOr(boolOr(stringEq(getConfigString(OBFUSCATE), "protected"), stringEq(getConfigString(OBFUSCATE), "full")),
-                  boolAnd(stringEq(getConfigString(OBFUSCATE), "encrypted"), Util.endsWith(fileName, ".moc")))
+    if not boolOr(boolOr(stringEq(getConfigString(OBFUSCATE), "protected"), stringEq(getConfigString(OBFUSCATE), "full")), stringEq(getConfigString(OBFUSCATE), "encrypted"))
```

Comment:
I see, I was taking it too far. I think this is what @perost meant and should be alright for now, only removing comments of protected variables. If comments remain w.r.t. #9119, we should replace all other calls to `escapeCComments`.

```suggestion
    if stringEq(getConfigString(OBFUSCATE), "none")
```

Reply:
Yes, that's what I meant, we should check if it's "none" instead of checking if it's any of the values except for "none".